### DISCUSSION TITLE
How to modify item number?

### DISCUSSION THREAD

**Discussion by sogokt301072** [TriliumNext/Trilium#3101] [Q&A]
As the image shows
There is an item which is number 3
A table is inserted as a new paragraph under the number 3 item 
Under the table, I want to have a new item, which is number 4
However, there is no way for me to insert the 4th item or to modify the item number
Is there any solution for it?
![image](https://user-images.githubusercontent.com/50332838/187110669-c8824883-f269-4385-b0b2-54c335116de6.png)

**Comment by zadam** [comment]
I don't think there is at the moment. Lists are numbered automatically but can't contain block elements such as tables. But IIRC CKEditor team is currently working on adding support for arbitrary content within list elements which should allow what you want.
